src/style.css

```diff
@@ -0,0 +1,93 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 400;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Avenir, serif;
+}
+
+.search-form1 {
+  margin-top: 20px;
+}
+
+.icon {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 34px;
+  top: 45px;
+}
+
+.icon--small {
+  width: 11px;
+  height: 11px;
+  left: 21px;
+  top: 125px;
+}
+
+.input {
+  /* box-sizing: border-box; */
+  margin-bottom: 20px;
+  padding: 0 62px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: inherit;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  width: 100%;
+  height: 70px;
+  outline: none;
+}
```

Comment:
Try to follow some CSS propety order rules https://medium.com/swlh/better-ways-to-organise-css-properties-9a066e7ded62